apps/server/test/graphql/Task.test.ts

```diff
@@ -770,4 +770,124 @@ describe("Task GraphQL mutations", () => {
       });
     });
   });
+
+  describe("addSubTask", () => {
+    it("creates a sub task with valid parent task Id but no item", async () => {
+      const { task } = await new Factory().newTask().run();
+
+      const res = await graphql({
+        query: /* GraphQL */ `
+          mutation AddSubTaskMutation($parentTaskId: ID!, $title: String!) {
+            addSubTask(input: { parentTaskId: $parentTaskId, title: $title }) {
+              id
+              date
+              title
+              status
+              item {
+                id
+              }
+            }
+          }
+        `,
```

Comment:
You can move the GraphQL out and re-use it in each test:
```suggestion
  describe("addSubTask", () => {
    const query = /* GraphQL */ `
      mutation AddSubTaskMutation($parentTaskId: ID!, $title: String!) {
        addSubTask(input: { parentTaskId: $parentTaskId, title: $title }) {
          id
          date
          title
          status
          item {
            id
          }
        }
      }
    `;
        
    it("creates a sub task with valid parent task Id but no item", async () => {
      const { task } = await new Factory().newTask().run();

      const res = await graphql({
        query,
```